MergerLogic/ImageProcessing/TileScaler.cs

```diff
@@ -19,8 +19,7 @@ public TileScaler(IMetricsProvider metricsProvider)
 
         public MagickImage? Upscale(MagickImage baseImage, Tile baseTile, Coord targetCoords)
         {
-            var upscaleStopWatch = new Stopwatch();
-            upscaleStopWatch.Start();
+            var upscaleStopWatch = Stopwatch.StartNew();
```

Comment:
This variable only has this function as it's scope.
Please rename to something like `stopwatch`.

Reply:
done